requirements.txt

```diff
@@ -1,2 +1,2 @@
-coincurve>=15.0,<19
+coincurve>=15.0,<=19
```

Comment:
```
coincurve<=19,>=15.0 not satisfied by version 19.0.1
```
I guess you meant `<20` instead of `<=19`?

Reply:
coincurve 19 is yanked

Reply:
I don't get what you mean.  19.0.0 is marked as yanked.  I was mentioning 19.0.1 which is not.
https://pypi.org/project/coincurve/#history

Reply:
I just added why <=19 is not working